src/Uno.UI.FluentTheme/Resources/PriorityDefault/NavigationView_rs2_themeresources.xaml

```diff
@@ -8,8 +9,8 @@
 
     <ResourceDictionary.ThemeDictionaries>
         <ResourceDictionary x:Key="Default">
-            <StaticResource x:Key="NavigationViewDefaultPaneBackground" ResourceKey="SystemControlTransientAcrylicElementBrush" />
-            <StaticResource x:Key="NavigationViewTopPaneBackground" ResourceKey="SystemControlTransientAcrylicElementBrush" />
+            <StaticResource x:Key="NavigationViewDefaultPaneBackground" ResourceKey="SystemControlAcrylicElementBrush" />
```

Comment:
Can you add a comment at each of these changes indicating it's an Uno workaround, to help manage upstream changes.